nixos/modules/system/boot/loader/grub/grub.nix

```diff
@@ -8,13 +8,15 @@ let
 
   efi = config.boot.loader.efi;
 
-  realGrub = if cfg.version == 1 then pkgs.grub
-    else if cfg.zfsSupport then pkgs.grub2.override { zfsSupport = true; }
+  targetPkgs = if cfg.forcei686 then pkgs.pkgsi686Linux else pkgs;
```

Comment:
This is okay - although maybe a comment saying what this is would be more clear.

Reply:
Actually grubPkgs makes sense for consistencyt.